### ISSUE TITLE
How can I use this code to do predict on custom dataset without ground truth labels? 

### ISSUE THREAD

**Issue by PacteraKun** [kjunelee/MetaOptNet#16]
Thank you for your great work. I wonder if you can help me to figure out how to do predict on dataset without ground truth labels. The label space of this dataset might be the same as that of training dataset. Thanks.

**Comment by kjunelee** [comment]
I think this might be useful for your problem: Meta-Learning Update Rules for Unsupervised Representation Learning (https://openreview.net/forum?id=HkNDsiC9KQ)

**Comment by PacteraKun** [comment]
Thanks for your advices. I will try it out.
